src/components/auth/ForgotPasswordComp.jsx

```diff
@@ -3,12 +3,26 @@ import { connect } from "react-redux";
 import PropTypes from "prop-types";
 import { SubmitButton, TextInput } from "../../views/Form/index";
 import { handleForgotPassword } from "../../redux/actions/ForgotPasswordAction";
+import { clearState } from "../../redux/actions/common";
 
 export class ForgotPassword extends Component {
   state = {
     user: { email: "", client_url: window.location.href }
   };
 
+  componentDidMount() {
+    const { dispatch } = this.props;
+    const registerModal = document.querySelector(
+      "#ahResetPasswordModalCloseButton"
+    );
```

Comment:
Empty line please before this if statements

Reply:
All done

Reply:
added the empty line